Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -183,13 +193,8 @@ void GeneralWidget::LoadSettings()
   m_custom_aspect_height->setHidden(!is_custom_aspect_ratio);
 }
 
-void GeneralWidget::SaveSettings()
+void GeneralWidget::BackendWarning()
```

Comment:
Small nit, can we rename this to `OnBackendSelectionChanged()` or something else?  `BackendWarning()` implies to me that it is only going to issue a warning but that is only a possibility (and it can even change the backend)